src/com/sun/ts/tests/ejb32/lite/timer/schedule/expire/Client.java

```diff
@@ -463,7 +468,13 @@ public void leapYears() {
       expectedNextTimeout = DateUtils.setMonths(expectedNextTimeout,
           Calendar.FEBRUARY);
       Timer timer = createTimer(exp);
-      verifyNextTimeout(expectedNextTimeout, timer);
+
+      // Use custom ignorable millis to accommodate the hour difference between
+      // the test timer's date (e.g., Feb 29) and execution date (e.g., March)
+      // due to daylight time switch in certain regions.
+      // See https://github.com/eclipse-ee4j/jakartaee-tck/issues/163
+      verifyNextTimeout(expectedNextTimeout, timer,
+              DateUtils.MILLIS_PER_HOUR + DateUtils.MILLIS_PER_MINUTE);
```

Comment:
@chengfang would it be better to change the tests to be consistent about either using US Standard Time or US Daylight Saving Time, but not both?

Reply:
Better still, use UTC.

Reply:
These tests are testing leap years, so my thinking was to verify year, month and day, while be loose on hours and minutes, without complicating the tests with timezone. If we want to add a consistent timezone to this set of tests, I can give modify the fix and test how it goes.